app/assets/javascripts/alchemy/templates/node_folder.hbs

```diff
@@ -1,3 +1,3 @@
-<a class="node_folder" data-node-id="{{ node.id }}">
+<a class="node_folder" data-node-id="{{ node.id }}" data-record-type="{{ node.type }}">
```

Comment:
Shouldn't this be renamed to `data-record-id` according to the click handler below?

Reply:
@tvdeyen My bad, updated it.